protocol/x/sending/keeper/transfer_test.go

```diff
@@ -462,3 +463,111 @@ func TestProcessWithdrawFromSubaccount(t *testing.T) {
 		})
 	}
 }
+
+func TestSendFromModuleToAccount(t *testing.T) {
+	testDenom := "TestSendFromModuleToAccount/Coin"
+	testModuleName := "bridge"
+
+	tests := map[string]struct {
+		// Setup.
+		initialModuleBalance int64
+		balanceToSend        int64
+		recipientAddress     string
+
+		// Expectations
+		expectedErrContains string
+	}{
+		"Success - send to user account": {
+			initialModuleBalance: 1000,
+			balanceToSend:        100,
+			recipientAddress:     constants.AliceAccAddress.String(),
+		},
+		"Success - send to module account": {
+			initialModuleBalance: 1000,
+			balanceToSend:        100,
+			recipientAddress:     authtypes.NewModuleAddress("community_treasury").String(),
+		},
```

Comment:
Can we add a test case for msg fails ValidateBasic?

Reply:
Would it be possible to also add a test for sending from an nonexistent account with a valid formatted address?

Reply:
Will try to add both!

Reply:
Can you add a test that sends funds from a module account to itself? Assume this causes an error, let's check

Reply:
it doesn't cause an error. It subtracts funds from the module account and then adds funds to it. Adding that test doesn't seem quite necessary to me and it also requires changing the way unit tests are currently written. Let me know your thoughts